File: backend/src/main/java/com/project/backend/domain/follow/dto/FollowResponseDto.java
package com.project.backend.domain.follow.dto;

/**
 * 팔로우 응답 DTO(record)
 * author: 이원재
 * date: 2025.01.31
 */
public record FollowResponseDto(
        String memberId,
        String nickname,
        Long followerCount,
        Long followingCount
) {}

File: backend/src/main/java/com/project/backend/domain/follow/dto/FollowResponseDto.kt
package com.project.backend.domain.follow.dto;

/**
 * 팔로우 응답 DTO(record)
 * author: 이원재
 * date: 2025.01.31
 */
data class FollowResponseDto(
        val username: String,
        val nickname: String,
        val followerCount: Long,
        val followingCount: Long
) {}
